components/config/Cargo.toml

```diff
@@ -30,5 +30,5 @@ servo_config_plugins = { path = "../config_plugins" }
 [dev-dependencies]
 std_test_override = { path = "../std_test_override" }
 
-[target.'cfg(not(any(target_os = "android", feature = "uwp")))'.dependencies]
```

Comment:
This caused a new warning:

```
warning: /home/simon/servo1/components/config/Cargo.toml: Found `feature = ...` in `target.'cfg(...)'.dependencies`. This key is not supported for selecting dependencies and will not work as expected. Use the [features] section instead: https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section
```

As far as I know it’s not possible to make a Cargo feature disable a dependency.